spec/support/http.rb

```diff
@@ -18,6 +18,20 @@ def self.testable?(mounted, requested)
         (mounted == requested) ||
           (mounted == "get" && requested == "head")
       end
+
+      def self.headers(expected)
```

Comment:
```suggestion
      def self.headers(headers_hash)
```

Having "expected" as an arg name here feels a bit confusing because that's a name we typically use when making test assertions/expectations. Here we're just conditionally converting the a headers hash into a different kind of hash-like object. So I think it's best to avoid the misleading argument name.

Reply:
It would also be good if we added a line of docs above this method, explaining why we have it and what it does.

Reply:
Another thought: I'm not in love with the quite verbose full `RSpec::Support::HTTP.headers` method name we have to pepper all over our spec files.

What if we:

1. moved this method onto another module, say `RSpec::Support::Rack`
2. called the method `rack_headers`
3. and then did an RSpec `config.include RSpec::Support::Rack`
4. allowing us to do a plain `rack_headers(...)` method call throughout our specs?

That feels like a lot less noise in the tests, helping us keep the focus on the actual header values that we're passing to the method.

Would you be up for giving that a go?

Reply:
great suggestion, happy to make the change